supabase/supabase.service.ts

```diff
@@ -0,0 +1,37 @@
+// supabase.service.ts
+
+import { Injectable } from '@nestjs/common';
+import { createClient, SupabaseClient } from '@supabase/supabase-js';
+
+@Injectable()
+export class SupabaseService {
+  private supabase: SupabaseClient;
+
+  constructor() {
+    const supabaseURL = process.env.SUPABASE_URL;
+    const supabaseAPIKey = process.env.SUPABASE_URL;
```

Comment:
shouldnt supabaseAPIKey be the supabase anon key? not sure bakit di nag eerror but either way can you set this to the anon key